api/proto/common/v2/common_v2.proto

```diff
@@ -3,21 +3,39 @@ package common.v2;
 import "common/common.proto";
 option go_package = "github.com/Layr-Labs/eigenda/api/grpc/common/v2";
 
-
+// BlobHeader contains the information needed to disperse a blob to the EigenDA network.
```

Comment:
IMO this one-liner could be more about what does BlobHeader describe (definitive properties of a blob), rather than where it's used

Reply:
Changed to the following:

```
// BlobHeader contains the information describing a blob and the way it is to be dispersed.
```

TODO commit